apps/antalmanac/src/components/RightPane/SectionTable/CourseInfoButton.tsx

```diff
@@ -86,7 +87,7 @@ function CourseInfoButton({
                     }
                 }}
             >
-                {text}
+                <span style={{ whiteSpace: 'nowrap', overflow: 'hidden' }}>{text}</span>
```

Comment:
```suggestion
                <span style={{ whiteSpace: 'nowrap' }}>{text}</span>
```
Ideally, these buttons shouldn't ever overflow, and given the current CSS, I don't think they should